### PR TITLE
Style all standard select as the text input in the backoffice

### PR DESCRIPTION
**Pull request by seballot**
### Before

image

### After

image

### How to test

As an admin manager, test the look and feel of as many as possible select input in the admin section.

### CODE REVIEW

File: app/assets/stylesheets/admin/shared/forms.scss [hunk 1]
```diff
@@ -283,3 +283,8 @@ fieldset {
 .form-buttons {
   text-align: center;
 }
+
+select {
+  @extend input[type="text"];
+  background-color: white;
+}
```
Comment (jibees) [inline app/assets/stylesheets/admin/shared/forms.scss:290]: I'd add a new line here please :pray:

**Review by jibees — APPROVED** [review]
LGMT.

Please edit the _How to test?_ section. Something like this I guess:

As an admin manager, test the look and feel of as many as possible select input in the admin section.

**Review by mkllnk — APPROVED** [review]
Nice. :sparkles: 

> As an admin manager, test the look and feel of as many as possible select input in the admin section.

It would have been nice to list some pages.

**Comment by filipefurtad0** [comment]
Hey @seballot ,
Nice! Thanks again :tada: 

This is also a case in which I'd specifically ping product, as this is not PR is not connected to any issue and does tweak design a bit. I personally feel some aesthetically coherence is gained but perhaps some contrast is lost, which may compromise readability - but I'd say @openfoodfoundation/train-drivers-product-owners have the call:

image

Added the feedback label :+1:

**Comment by RachL** [comment]
This page will be soon completely re-done as part of Back Office UI Uplift project. I don't mind moving this one forward in the meantime. The contrast was focusing on non-important fields anyway :D

**Pull request merged**